frontend/app-development/hooks/queries/useAppLibQuery.ts

```diff
@@ -0,0 +1,9 @@
+import { useQuery } from '@tanstack/react-query';
+import { getAppLibVersion } from 'app-shared/api/queries';
+import { QueryKey } from 'app-shared/types/QueryKey';
+
+export const useAppLibQuery = (org: string, repo: string) => {
+  return useQuery<{ version: string }>([QueryKey.AppLibVersion, org, repo], () =>
```

Comment:
Can you write this the same way as the new API from React Query? 😊 

```suggestion
  return useQuery<{ version: string }>({ queryKey: [QueryKey.AppLibVersion, org, repo],  queryFn: () =>
```